frontend/src/libs/components/meditation-player/components/audio-player/audio-player.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback, useEffect, useRef, useState } from '#libs/hooks/hooks.js';
+
+import { START_TIME, STEP } from '../../libs/constants.js';
+import { type Meditation } from '../../libs/types.js';
```

Comment:
Maybe it's better to create subfolder for types and constants and create separate file for type?

Reply:
Please, explain me, what do you mean ? 
I created a libs folder with types in meditation-player folder.

Reply:
I mean that in our project we have such structure like on screenshot where we have `types `forlder and there we have `types.ts` file that exports all types in this folder
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111736110/59f8e9b2-1928-4547-98c8-d7b17500c86c)

Reply:
Done